[BUG] Meteor Client Crashing in Latest Releases

### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

0.24.1

### Operating System

Windows 11

### Minecraft Version

1.21.4

### Describe the bug

I wish to pair Meteor Client with LB to make use of Meteors auto-web module. Unfortunately, Meteor Client crashes on startup, making this pairing impossible.


### Steps to reproduce

- Download Meteor 1.21.4
- Add it to the mods list
- Launch

### Client Log

```shell
Loading version manifest...Downloading recommended mod LiquidBounceDownloading recommended mod Fabric APIDownloading recommended mod Fabric KotlinDownloading recommended mod NoChatReportsDownloading recommended mod Reese's Sodium OptionsDownloading recommended mod Sodium ExtraDownloading recommended mod SodiumDownloading recommended mod ViaFabricPlusCopied custom mod meteor-client-1.21.4-4Loading version profile...Downloading inherited version 1.21.4...Launching 6f02ff611c03a3292130f8bcb6be564ee6e93f91...Setting up...Determined OS to be windows 10.0.26100Checking for JRE...Java Path: "C:\\Users\\--------\\AppData\\Roaming\\CCBlueX\\LiquidLauncher\\data\\runtimes\\temurin_21\\jdk-21.0.5 11-jre\\bin\\javaw.exe"Client JAR local hash: a7e5a6024bfd3cd614625aa05629adf760020304, remote: a7e5a6024bfd3cd614625aa05629adf760020304Checking libraries...Library org.apache.logging.log4j:log4j-core:2.24.1 already exists and SHA1 matches.Library org.lwjgl:lwjgl-glfw:3.3.3 already exists and SHA1 matches.Library org.lwjgl:lwjgl-jemalloc:3.3.3 already exists and SHA1 matches.Library org.lwjgl:lwjgl-opengl:3.3.3:natives-windows-arm64 already exists and SHA1 matches.Library org.lwjgl:lwjgl-glfw:3.3.3:natives-windows already exists and SHA1 matches.Library org.ow2.asm:asm-commons:9.7.1 already exists and SHA1 matches.Library org.lwjgl:lwjgl-openal:3.3.3:natives-windows-arm64 already exists and SHA1 matches.Library com.mojang:patchy:2.2.10 already exists and SHA1 matches.Library net.minidev:json-smart:2.5.1 already exists and SHA1 matches.Library com.google.guava:failureaccess:1.0.2 already exists and SHA1 matches.Library commons-io:commons-io:2.17.0 already exists and SHA1 matches.Library org.lwjgl:lwjgl-tinyfd:3.3.3 already exists and SHA1 matches.Library io.netty:netty-transport-classes-epoll:4.1.115.Final already exists and SHA1 matches.Library com.nimbusds:lang-tag:1.7 already exists and SHA1 matches.Library com.nimbusds:oauth2-oidc-sdk:11.18 already exists and SHA1 matches.Library net.java.dev.jna:jna:5.15.0 already exists and SHA1 matches.Library org.lwjgl:lwjgl-tinyfd:3.3.3:natives-windows-arm64 already exists and SHA1 matches.Library io.netty:netty-common:4.1.115.Final already exists and SHA1 matches.Library org.lwjgl:lwjgl-opengl:3.3.3 already exists and SHA1 matches.Library net.fabricmc:fabric-loader:0.16.9 already exists and SHA1 matches.Library org.lwjgl:lwjgl:3.3.3:natives-windows-arm64 already exists and SHA1 matches.Library org.jcraft:jorbis:0.0.17 already exists and SHA1 matches.Library com.fasterxml.jackson.core:jackson-annotations:2.13.4 already exists and SHA1 matches.Library com.microsoft.azure:msal4j:1.17.2 already exists and SHA1 matches.Library com.mojang:jtracy:1.0.29 already exists and SHA1 matches.Library org.apache.logging.log4j:log4j-slf4j2-impl:2.24.1 already exists and SHA1 matches.Library org.lwjgl:lwjgl-glfw:3.3.3:natives-windows-arm64 already exists and SHA1 matches.Library org.lwjgl:lwjgl-glfw:3.3.3:natives-windows-x86 already exists and SHA1 matches.Library org.lwjgl:lwjgl-stb:3.3.3:natives-windows-arm64 already exists and SHA1 matches.Library commons-logging:commons-logging:1.3.4 already exists and SHA1 matches.Library org.lwjgl:lwjgl-tinyfd:3.3.3:natives-windows-x86 already exists and SHA1 matches.Library org.lz4:lz4-java:1.8.0 already exists and SHA1 matches.Library org.ow2.asm:asm:9.7.1 already exists and SHA1 matches.Library org.apache.httpcomponents:httpclient:4.5.14 already exists and SHA1 matches.Library com.mojang:jtracy:1.0.29:natives-windows already exists and SHA1 matches.Library org.lwjgl:lwjgl-opengl:3.3.3:natives-windows already exists and SHA1 matches.Library io.netty:netty-codec:4.1.115.Final already exists and SHA1 matches.Library com.mojang:datafixerupper:8.0.16 already exists and SHA1 matches.Library com.github.oshi:oshi-core:6.6.5 already exists and SHA1 matches.Library org.lwjgl:lwjgl-jemalloc:3.3.3:natives-windows already exists and SHA1 matches.Library com.mojang:brigadier:1.3.10 already exists and SHA1 matches.Library com.nimbusds:content-type:2.3 already exists and SHA1 matches.Library org.joml:joml:1.10.8 already exists and SHA1 matches.Library com.mojang:text2speech:1.17.9 already exists and SHA1 matches.Library org.lwjgl:lwjgl:3.3.3 already exists and SHA1 matches.Library commons-codec:commons-codec:1.17.1 already exists and SHA1 matches.Library io.netty:netty-handler:4.1.115.Final already exists and SHA1 matches.Library io.netty:netty-resolver:4.1.115.Final already exists and SHA1 matches.Library org.ow2.asm:asm-analysis:9.7.1 already exists and SHA1 matches.Library org.lwjgl:lwjgl-opengl:3.3.3:natives-windows-x86 already exists and SHA1 matches.Library com.mojang:blocklist:1.0.10 already exists and SHA1 matches.Library org.lwjgl:lwjgl-openal:3.3.3:natives-windows already exists and SHA1 matches.Library org.lwjgl:lwjgl-stb:3.3.3:natives-windows already exists and SHA1 matches.Library com.mojang:logging:1.5.10 already exists and SHA1 matches.Library net.java.dev.jna:jna-platform:5.15.0 already exists and SHA1 matches.Library org.apache.httpcomponents:httpcore:4.4.16 already exists and SHA1 matches.Library com.ibm.icu:icu4j:76.1 already exists and SHA1 matches.Library io.netty:netty-transport:4.1.115.Final already exists and SHA1 matches.Library org.lwjgl:lwjgl-tinyfd:3.3.3:natives-windows already exists and SHA1 matches.Library org.apache.logging.log4j:log4j-api:2.24.1 already exists and SHA1 matches.Library org.apache.commons:commons-lang3:3.17.0 already exists and SHA1 matches.Library com.fasterxml.jackson.core:jackson-core:2.13.4 already exists and SHA1 matches.Library org.lwjgl:lwjgl-stb:3.3.3:natives-windows-x86 already exists and SHA1 matches.Library org.lwjgl:lwjgl-freetype:3.3.3:natives-windows already exists and SHA1 matches.Library org.lwjgl:lwjgl-freetype:3.3.3 already exists and SHA1 matches.Library org.lwjgl:lwjgl-freetype:3.3.3:natives-windows-x86 already exists and SHA1 matches.Library org.lwjgl:lwjgl-jemalloc:3.3.3:natives-windows-arm64 already exists and SHA1 matches.Library org.ow2.asm:asm-util:9.7.1 already exists and SHA1 matches.Library it.unimi.dsi:fastutil:8.5.15 already exists and SHA1 matches.Library org.apache.commons:commons-compress:1.27.1 already exists and SHA1 matches.Library io.netty:netty-buffer:4.1.115.Final already exists and SHA1 matches.Library net.sf.jopt-simple:jopt-simple:5.0.4 already exists and SHA1 matches.Library org.lwjgl:lwjgl-stb:3.3.3 already exists and SHA1 matches.Library io.netty:netty-transport-native-unix-common:4.1.115.Final already exists and SHA1 matches.Library org.lwjgl:lwjgl:3.3.3:natives-windows already exists and SHA1 matches.Library com.github.stephenc.jcip:jcip-annotations:1.0-1 already exists and SHA1 matches.Library com.nimbusds:nimbus-jose-jwt:9.40 already exists and SHA1 matches.Library org.slf4j:slf4j-api:2.0.16 already exists and SHA1 matches.Library org.lwjgl:lwjgl-openal:3.3.3:natives-windows-x86 already exists and SHA1 matches.Library org.lwjgl:lwjgl-freetype:3.3.3:natives-windows-arm64 already exists and SHA1 matches.Library net.minidev:accessors-smart:2.5.1 already exists and SHA1 matches.Library org.lwjgl:lwjgl-openal:3.3.3 already exists and SHA1 matches.Library org.lwjgl:lwjgl:3.3.3:natives-windows-x86 already exists and SHA1 matches.Library com.fasterxml.jackson.core:jackson-databind:2.13.4.2 already exists and SHA1 matches.Library com.google.code.gson:gson:2.11.0 already exists and SHA1 matches.Library org.ow2.asm:asm-tree:9.7.1 already exists and SHA1 matches.Library org.lwjgl:lwjgl-jemalloc:3.3.3:natives-windows-x86 already exists and SHA1 matches.Library net.fabricmc:sponge-mixin:0.15.4 mixin.0.8.7 already exists and SHA1 matches.Library com.mojang:authlib:6.0.57 already exists and SHA1 matches.Library net.fabricmc:intermediary:1.21.4 already exists and SHA1 matches.Library com.google.guava:guava:33.3.1-jre already exists and SHA1 matches.Checking assets...Launching...Running...[20:18:45] [main/INFO]: Loading Minecraft 1.21.4 with Fabric Loader 0.16.9
[20:18:46] [ForkJoinPool-1-worker-2/WARN]: Mod com_github_oryxel_cubeconverter uses the version 25bc6ce1b1 which isn't compatible with Loader's extended semantic version format (Could not parse version number component '25bc6ce1b1'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[20:18:46] [ForkJoinPool-1-worker-6/WARN]: Mod org_cloudburstmc_netty_netty-transport-raknet uses the version 1.0.0.CR3-SNAPSHOT which isn't compatible with Loader's extended semantic version format (Could not parse version number component 'CR3'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[20:18:46] [main/INFO]: Loading 144 mods:
	- fabric-api 0.111.0 1.21.4
	   |-- fabric-api-lookup-api-v1 1.6.84 7feeb73304
	   |-- fabric-biome-api-v1 15.0.4 466a140b04
	   |-- fabric-block-api-v1 1.0.31 7feeb73304
	   |-- fabric-block-view-api-v2 1.0.19 7feeb73304
	   |-- fabric-blockrenderlayer-v1 2.0.7 7feeb73304
	   |-- fabric-client-tags-api-v1 1.1.27 7feeb73304
	   |-- fabric-command-api-v1 1.2.61 f71b366f04
	   |-- fabric-command-api-v2 2.2.40 e496eb1504
	   |-- fabric-commands-v0 0.2.78 df3654b304
	   |-- fabric-content-registries-v0 9.1.12 7feeb73304
	   |-- fabric-convention-tags-v1 2.1.6 7f945d5b04
	   |-- fabric-convention-tags-v2 2.9.6 466a140b04
	   |-- fabric-crash-report-info-v1 0.3.6 7feeb73304
	   |-- fabric-data-attachment-api-v1 1.4.3 9aea556b04
	   |-- fabric-data-generation-api-v1 22.1.2 466a140b04
	   |-- fabric-dimensions-v1 4.0.10 7feeb73304
	   |-- fabric-entity-events-v1 2.0.11 60fccc7604
	   |-- fabric-events-interaction-v0 4.0.1 a4eebcf004
	   |-- fabric-game-rule-api-v1 1.0.62 7feeb73304
	   |-- fabric-item-api-v1 11.1.12 203e6b2304
	   |-- fabric-item-group-api-v1 4.1.21 7feeb73304
	   |-- fabric-key-binding-api-v1 1.0.56 7feeb73304
	   |-- fabric-keybindings-v0 0.2.54 df3654b304
	   |-- fabric-lifecycle-events-v1 2.5.3 bf2a60eb04
	   |-- fabric-loot-api-v2 3.0.31 3f89f5a504
	   |-- fabric-loot-api-v3 1.0.19 203e6b2304
	   |-- fabric-message-api-v1 6.0.24 7feeb73304
	   |-- fabric-model-loading-api-v1 4.1.0 7b6b225304
	   |-- fabric-networking-api-v1 4.3.8 cc0fa2fe04
	   |-- fabric-object-builder-api-v1 18.0.5 e604fe7f04
	   |-- fabric-particles-v1 4.0.13 7feeb73304
	   |-- fabric-recipe-api-v1 8.0.3 fa62a02304
	   |-- fabric-renderer-indigo 2.0.1 84404cdd04
	   |-- fabric-rendering-data-attachment-v1 0.3.57 73761d2e04
	   |-- fabric-rendering-v1 10.1.0 7e7568bf04
	   |-- fabric-resource-conditions-api-v1 5.0.13 203e6b2304
	   |-- fabric-screen-api-v1 2.0.37 7feeb73304
	   |-- fabric-screen-handler-api-v1 1.3.106 7feeb73304
	   |-- fabric-sound-api-v1 1.0.32 7feeb73304
	   |-- fabric-transfer-api-v1 5.4.6 1885ad7404
	   \-- fabric-transitive-access-wideners-v1 6.3.2 56e78b9b04
	- fabric-language-kotlin 1.13.0 kotlin.2.1.0
	   |-- org_jetbrains_kotlin_kotlin-reflect 2.1.0
	   |-- org_jetbrains_kotlin_kotlin-stdlib 2.1.0
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk7 2.1.0
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk8 2.1.0
	   |-- org_jetbrains_kotlinx_atomicfu-jvm 0.26.1
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm 1.9.0
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-jdk8 1.9.0
	   |-- org_jetbrains_kotlinx_kotlinx-datetime-jvm 0.6.1
	   |-- org_jetbrains_kotlinx_kotlinx-io-bytestring-jvm 0.6.0
	   |-- org_jetbrains_kotlinx_kotlinx-io-core-jvm 0.6.0
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm 1.7.3
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-core-jvm 1.7.3
	   \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.7.3
	- fabricloader 0.16.9
	   \-- mixinextras 0.4.1
	- java 21
	- liquidbounce 0.24.1
	   |-- com_github_ccbluex_discordipc 4.0.0
	   |-- com_github_ccbluex_mc-authlib 1.4.1
	   |-- com_github_ccbluex_netty-httpserver 2.1.1
	   |-- com_google_code_gson_gson 2.10.1
	   |-- com_kohlschutter_junixsocket_junixsocket-common 2.8.3
	   |-- com_kohlschutter_junixsocket_junixsocket-native-common 2.8.3
	   |-- com_squareup_okhttp3_okhttp 5.0.0-alpha.14
	   |-- com_squareup_okio_okio-jvm 3.9.0
	   |-- com_thealtening_api_api 4.1.0
	   |-- com_vdurmont_semver4j 3.1.0
	   |-- commons-codec_commons-codec 1.17.1
	   |-- commons-io_commons-io 2.16.1
	   |-- io_netty_netty-codec-socks 4.1.97
	   |-- io_netty_netty-handler 4.1.97
	   |-- io_netty_netty-handler-proxy 4.1.97
	   |-- io_netty_netty-resolver 4.1.97
	   |-- io_netty_netty-transport-native-unix-common 4.1.97
	   |-- mcef 1.3.2-1.21.4
	   |-- net_fabricmc_tiny-mappings-parser 0.3.0 build.17
	   |-- org_ahocorasick_ahocorasick 0.6.3
	   |-- org_apache_commons_commons-compress 1.27.1
	   |-- org_apache_commons_commons-exec 1.3
	   |-- org_apache_commons_commons-lang3 3.16.0
	   |-- org_apache_logging_log4j_log4j-api 2.23.1
	   |-- org_apache_logging_log4j_log4j-core 2.23.1
	   |-- org_apache_tika_tika-core 2.9.2
	   |-- org_graalvm_js_js-language 24.0.2
	   |-- org_graalvm_polyglot_polyglot 24.0.2
	   |-- org_graalvm_regex_regex 24.0.2
	   |-- org_graalvm_sdk_collections 24.0.2
	   |-- org_graalvm_sdk_jniutils 24.0.2
	   |-- org_graalvm_sdk_nativeimage 24.0.2
	   |-- org_graalvm_sdk_word 24.0.2
	   |-- org_graalvm_shadowed_icu4j 24.0.2
	   |-- org_graalvm_shadowed_json 24.0.2
	   |-- org_graalvm_tools_chromeinspector-tool 24.0.2
	   |-- org_graalvm_tools_coverage-tool 24.0.2
	   |-- org_graalvm_tools_dap-tool 24.0.2
	   |-- org_graalvm_tools_insight-heap-tool 24.0.2
	   |-- org_graalvm_tools_insight-tool 24.0.2
	   |-- org_graalvm_tools_lsp-tool 24.0.2
	   |-- org_graalvm_tools_lsp_api 24.0.2
	   |-- org_graalvm_tools_profiler-tool 24.0.2
	   |-- org_graalvm_truffle_truffle-api 24.0.2
	   |-- org_graalvm_truffle_truffle-compiler 24.0.2
	   |-- org_graalvm_truffle_truffle-runtime 24.0.2
	   \-- org_slf4j_slf4j-api 2.0.10
	- meteor-client 1.21.4-4
	   \-- fabric-resource-loader-v0 3.0.11 b1caf1e904
	- minecraft 1.21.4
	- nochatreports 1.21.4-v2.11.0
	- reeses-sodium-options 1.8.2 mc1.21.4
	- sodium 0.6.6 mc1.21.4
	   |-- fabric-api-base 0.4.54 b47eab6b04
	   |-- fabric-renderer-api-v1 5.0.2 84404cdd04
	   \-- fabric-rendering-fluids-v1 3.1.19 7feeb73304
	- sodium-extra 0.6.1 mc1.21.4
	- viafabricplus 4.0.1
	   |-- com_github_oryxel_cubeconverter 25bc6ce1b1
	   |-- com_google_code_findbugs_jsr305 3.0.2
	   |-- com_vdurmont_semver4j 3.1.0
	   |-- com_viaversion_viaaprilfools-common 4.0.0
	   |-- com_viaversion_viabackwards-common 5.2.1
	   |-- com_viaversion_vialoader 4.0.0
	   |-- com_viaversion_viaversion-common 5.2.1
	   |-- de_florianmichael_classic4j 2.1.1-SNAPSHOT
	   |-- fabric-registry-sync-v0 6.1.0 9ba00ebb04
	   |-- io_jsonwebtoken_jjwt-api 0.12.6
	   |-- io_jsonwebtoken_jjwt-gson 0.12.6
	   |-- io_jsonwebtoken_jjwt-impl 0.12.6
	   |-- io_netty_netty-codec-http 4.1.116
	   |-- net_jodah_expiringmap 0.5.10
	   |-- net_lenni0451_commons_httpclient 1.6.0
	   |-- net_lenni0451_mcping 1.4.2
	   |-- net_lenni0451_mcstructs-bedrock_forms 1.2.2
	   |-- net_lenni0451_mcstructs-bedrock_text 1.2.2
	   |-- net_lenni0451_reflect 1.4.0
	   |-- net_raphimc_minecraftauth 4.1.1
	   |-- net_raphimc_viabedrock 0.0.14-20241220.132943-7
	   |-- net_raphimc_vialegacy 3.0.7
	   |-- org_cloudburstmc_netty_netty-transport-raknet 1.0.0.CR3-SNAPSHOT
	   |-- org_lz4_lz4-pure-java 1.8.0
	   |-- viafabricplus-api 4.0.1
	   |-- viafabricplus-api-legacy 4.0.1
	   \-- viafabricplus-visuals 4.0.1
[20:18:47] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.7 Source=file:/C:/Users/========/AppData/Roaming/CCBlueX/LiquidLauncher/data/libraries/net/fabricmc/sponge-mixin/0.15.4 mixin.0.8.7/sponge-mixin-0.15.4 mixin.0.8.7.jar Service=Knot/Fabric Env=CLIENT
[20:18:47] [main/INFO]: Compatibility level set to JAVA_21
[20:18:48] [main/INFO]: Loaded configuration file for Sodium: 43 options available, 0 override(s) found
[20:18:48] [main/INFO]: Loaded configuration file for Sodium Extra: 34 options available, 0 override(s) found
[20:18:49] [main/WARN]: Error loading class: baritone/command/defaults/ComeCommand (java.lang.ClassNotFoundException: baritone/command/defaults/ComeCommand)
[20:18:49] [main/WARN]: @Mixin target baritone.command.defaults.ComeCommand was not found meteor-client-baritone.mixins.json:ComeCommandMixin from mod meteor-client
[20:18:49] [main/WARN]: Error loading class: net/caffeinemc/mods/lithium/common/entity/movement/ChunkAwareBlockCollisionSweeper (java.lang.ClassNotFoundException: net/caffeinemc/mods/lithium/common/entity/movement/ChunkAwareBlockCollisionSweeper)
[20:18:50] [main/INFO]: Searching for graphics cards...
[20:18:50] [main/INFO]: Found graphics adapter: AdapterInfo{vendor=NVIDIA, description='NVIDIA GeForce GTX 1650', adapterType=0x00002313, openglIcdFilePath='C:\WINDOWS\System32\DriverStore\FileRepository\nvhm.inf_amd64_69784b7a3902e1a0\nvoglv64.dll', openglIcdVersion=32.0.15.5613}
[20:18:50] [main/INFO]: Found graphics adapter: AdapterInfo{vendor=INTEL, description='Intel(R) UHD Graphics 630', adapterType=0x0000232B, openglIcdFilePath='C:\WINDOWS\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_d60a3faa7932ec00\ig9icd64.dll', openglIcdVersion=27.20.100.8729}
[20:18:50] [main/WARN]: Sodium has applied one or more workarounds to prevent crashes or other issues on your system: [NVIDIA_THREADED_OPTIMIZATIONS_BROKEN]
[20:18:50] [main/WARN]: This is not necessarily an issue, but it may result in certain features or optimizations being disabled. You can sometimes fix these issues by upgrading your graphics driver.
[20:18:50] [main/INFO]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.4.1).
[20:18:55] [Datafixer Bootstrap/INFO]: 243 Datafixer optimizations took 1639 milliseconds
[20:19:07] [Render thread/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[20:19:07] [Render thread/INFO]: Setting user: EvidentGrass
[20:19:08] [Render thread/INFO]: [Indigo] Different rendering plugin detected; not applying Indigo.
[20:19:08] [Via-Mappingloader-0/INFO]: Loading block connection mappings ...
[20:19:08] [Via-Mappingloader-0/INFO]: Using FastUtil Long2ObjectOpenHashMap for block connections
[20:19:09] [Render thread/INFO]: Item Groups: [ Heads, Exploits, Containers ]
[20:19:09] [ForkJoinPool.commonPool-worker-1/INFO]: Loading translations...
[20:19:09] [ForkJoinPool.commonPool-worker-1/INFO]: Registering protocols...
[20:19:10] [Render thread/INFO]: Backend library: LWJGL version 3.3.3-snapshot
[20:19:10] [Render thread/INFO]: Modifying process environment to apply workarounds for the NVIDIA graphics driver...
[20:19:11] [ForkJoinPool.commonPool-worker-1/INFO]: Started resource pack HTTP server on http://127.0.0.1:3776/
[20:19:11] [ForkJoinPool.commonPool-worker-1/INFO]: ViaVersion detected lowest supported version by the proxy: c0.0.15a-1 (0)
[20:19:11] [ForkJoinPool.commonPool-worker-1/INFO]: Highest supported version by the proxy: 1.21.4 (769)
[20:19:11] [ForkJoinPool.commonPool-worker-1/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[20:19:14] [Render thread/INFO]: OpenGL Vendor: NVIDIA Corporation
[20:19:14] [Render thread/INFO]: OpenGL Renderer: NVIDIA GeForce GTX 1650/PCIe/SSE2
[20:19:14] [Render thread/INFO]: OpenGL Version: 3.2.0 NVIDIA 556.13
[20:19:14] [Render thread/INFO]: Modifying OpenGL context to apply workarounds for the NVIDIA graphics driver...
[20:19:14] [Render thread/INFO]: Enabling GL_DEBUG_OUTPUT_SYNCHRONOUS to force the NVIDIA driver to disable threadedcommand submission
[20:19:17] [Render thread/WARN]: @ModifyConstant conflict. Skipping meteor-client.mixins.json:PlayerListHudMixin from mod meteor-client->@ModifyConstant::modifyCount(J)J with priority 1000, already redirected by liquidbounce.mixins.json:minecraft.gui.MixinPlayerListHud from mod liquidbounce->@ModifyConstant::hookTabSize(J)J with priority 1337
---- Minecraft Crash Report ----
// Uh... Did I do that?

Time: 2025-01-16 20:19:17
Description: Initializing game

java.lang.RuntimeException: Mixin transformation of net.minecraft.class_355 failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:427)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at knot//net.minecraft.class_329.<init>(class_329.java:191)
	at knot//net.minecraft.class_310.<init>(class_310.java:652)
	at knot//net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202)
	at knot//meteordevelopment.meteorclient.asm.Asm$Transformer.transformClassBytes(Asm.java:103)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422)
	... 10 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Constant modifier method modifyCount(J)J in meteor-client.mixins.json:PlayerListHudMixin from mod meteor-client failed injection check, (0/1) succeeded. Scanned 0 target(s). Using refmap meteor-client-refmap.json
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:531)
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1490)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:752)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:330)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:246)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:437)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:418)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	... 14 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:427)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at knot//net.minecraft.class_329.<init>(class_329.java:191)
	at knot//net.minecraft.class_310.<init>(class_310.java:652)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:Advanced Windows 32 Base API:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		COMCTL32.dll:User Experience Controls Library:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:Crypto API32:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.26100.2894 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		CoreMessaging.dll:Microsoft CoreMessaging Dll:10.0.26100.1882 (WinBuild.160101.0800):Microsoft Corporation
		CoreUIComponents.dll:Microsoft Core UI Components Dll:10.0.26100.2454:Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.26100.2033 (WinBuild.160101.0800):Microsoft Corporation
		DEVOBJ.dll:Device Information Set DLL:10.0.26100.1150 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:DNS Client API DLL:10.0.26100.1591 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.26100.2033 (WinBuild.160101.0800):Microsoft Corporation
		GLU32.dll:OpenGL Utility Library DLL:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.26100.1150 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:IP Helper API:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Windows NT BASE API Client DLL:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Windows NT BASE API Client DLL:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		MMDevApi.dll:MMDevice API:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		MSCTF.dll:MSCTF Server DLL:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		Microsoft.Internal.WarpPal.dll
		MpOav.dll:IOfficeAntiVirus Module:4.18.24090.11 (26d75470953920f90fd0438874501e9d9a31e621):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.26100.2894 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		NvMessageBus.dll:NVIDIA Message Bus::NVIDIA Corporation
		OLEAUT32.dll:OLEAUT32.DLL:10.0.26100.1882 (WinBuild.160101.0800):Microsoft Corporation
		Ole32.dll:Microsoft OLE for Windows:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		OpenAL.dll:Main implementation library:1.23.1:
		POWRPROF.dll:Power Profile Helper DLL:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		Pdh.dll:Windows Performance Data Helper DLL:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		RPCRT4.dll:Remote Procedure Call Runtime:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		SETUPAPI.dll:Windows Setup API:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:Windows Shell Common Dll:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		UMPDC.dll:User Mode Power Dependency Coordinator:10.0.26100.1301 (WinBuild.160101.0800):Microsoft Corporation
		USER32.dll:Multi-User Windows USER API Client DLL:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corpoporation
		bcrypt.dll:Windows Cryptographic Primitives Library:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.26100.2033 (WinBuild.160101.0800):Microsoft Corporation
		cfgmgr32.dll:Configuration Manager DLL:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM  Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM for Windows:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		cryptnet.dll:Crypto Network Related API:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		d3d11.dll:Direct3D 11 Runtime:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		dcomp.dll:Microsoft DirectComposition Library:10.0.26100.2605 (WinBuild.160101.0800):Microsoft Corporation
		dinput8.dll:Microsoft DirectInput:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		directxdatabasehelper.dll:DirectXDatabaseHelper:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		drvstore.dll:Driver Store API:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		dwmapi.dll:Microsoft Desktop Window Manager API:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		dxcore.dll:DXCore:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		dxgi.dll:DirectX Graphics Infrastructure:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		extnet.dll:OpenJDK Platform binary:21.0.5.0:Eclipse Adoptium
		fwpuclnt.dll:FWP/IPsec User-Mode API:10.0.26100.2161 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		glfw.dll:GLFW 3.4.0 DLL:3.4.0:GLFW
		icm32.dll:Microsoft Color Management Module (CMM):10.0.26100.2314 (WinBuild.160101.0800):Microsoft Corporation
		igc64.dll:Intel Graphics Shader Compiler for Intel(R) Graphics Accelerator:27.20.100.8729:Intel Corporation
		igd10iumd64.dll:User Mode Driver for Intel(R) Graphics Technology:27.20.100.8729:Intel Corporation
		igdgmm64.dll:User Mode Driver for Intel(R) Graphics Technology:27.20.100.8729:Intel Corporation
		imagehlp.dll:Windows NT Image Helper:10.0.26100.1150 (WinBuild.160101.0800):Microsoft Corporation
		inputhost.dll:InputHost:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		java.dll:OpenJDK Platform binary:21.0.5.0:Eclipse Adoptium
		javaw.exe:OpenJDK Platform binary:21.0.5.0:Eclipse Adoptium
		jemalloc.dll
		jimage.dll:OpenJDK Platform binary:21.0.5.0:Eclipse Adoptium
		jli.dll:OpenJDK Platform binary:21.0.5.0:Eclipse Adoptium
		jna6766506607896067692.dll:JNA native library:7.0.2:Java(TM) Native Access (JNA)
		jsvml.dll:OpenJDK Platform binary:21.0.5.0:Eclipse Adoptium
		jvm.dll:OpenJDK 64-Bit server VM:21.0.5.0:Eclipse Adoptium
		kernel.appcore.dll:AppModel API Host:10.0.26100.1591 (WinBuild.160101.0800):Microsoft Corporation
		lwjgl.dll
		lwjgl_opengl.dll
		lwjgl_stb.dll
		m0.5.0:Eclipse Adoptium
		nlansp_c.dll:NLA Namespace Service Provider DLL:10.0.26100.1150 (WinBuild.160101.0800):Microsoft Corporation
		ntdll.dll:NT Layer DLL:10.0.26100.2454 (WinBuild.160101.0800):Microsoft Corporation
		ntmarta.dll:Windows NT MARTA provider:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		nvapi64.dll:NVIDIA NVAPI Library, Version 556.13 :32.0.15.5613:NVIDIA Corporation
		nvgpucomp64.dll:NVIDIA GPU Compiler Driver, Version 556.13 :32.0.15.5613:NVIDIA Corporation
		nvldumdx.dll:NVIDIA Driver Loader, Version 556.13 :32.0.15.5613:NVIDIA Corporation
		nvoglv64.dll:NVIDIA Compatible OpenGL ICD:32.0.15.5613:NVIDIA Corporation
		nvspcap64.dll:NVIDIA Game Proxy:3.20.0.118:NVIDIA Corporation
		nvwgf2umx.dll:NVIDIA D3D10 Driver, Version 556.13 :32.0.15.5613:NVIDIA Corporation
		opengl32.dll:OpenGL Client DLL:10.0.26100.1882 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:Windows System Performance Objects DLL:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		pfclient.dll Runtime Library:14.40.33810.0:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:21.0.5.0:Eclipse Adoptium
		win32u.dll:Win32u:10.0.26100.2605 (WinBuild.160101.0800):Microsoft Corporation
		windows.staterepositorycore.dll:Windows StateRepository API Core:10.0.26100.1882 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:Microsoft WinRT Storage API:10.0.26100.1457 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.26100.1882 (WinBuild.160101.0800):Microsoft Corporation
		wintypes.dll:Windows Base Types DLL:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		wldp.dll:Windows Lockdown Policy:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.26100.1882 (WinBuild.160101.0800):Microsoft Corporation
		wshunix.dll:AF_UNIX Winsock2 Helper DLL:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		xinput1_4.dll:Microsoft Common Controller API:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:21.0.5.0:Eclipse Adoptium
Stacktrace:
	at knot//net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- System Details --
Details:
	Minecraft Version: 1.21.4
	Minecraft Version ID: 1.21.4
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 21.0.5, Eclipse Adoptium
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Eclipse Adoptium
	Memory: 314597544 bytes (300 MiB) / 671088640 bytes (640 MiB) up to 4294967296 bytes (4096 MiB)
	CPUs: 8
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i5-9300H CPU @ 2.40GHz
	Identifier: Intel64 Family 6 Model 158 Stepping 10
	Microarchitecture: Coffee Lake
	Frequency (GHz): 2.40
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 8
	Graphics card #0 name: Intel(R) UHD Graphics 630
	Graphics card #0 vendor: Intel Corporation
	Graphics card #0 VRAM (MiB): 1024.00
	Graphics card #0 deviceId: VideoController1
	Graphics card #0 versionInfo: 27.20.100.8729
	Graphics card #1 name: NVIDIA GeForce GTX 1650
	Graphics card #1 vendor: NVIDIA
	Graphics card #1 VRAM (MiB): 4096.00
	Graphics card #1 deviceId: VideoController2
	Graphics card #1 versionInfo: 32.0.15.5613
	Memory slot #0 capacity (MiB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.67
	Memory slot #0 type: DDR4
	Virtual memory max (MiB): 18536.42
	Virtual memory used (MiB): 12206.71
	Swap memory total (MiB): 10503.24
	Swap memory used (MiB): 848.09
	Space in storage for jna.tmpdir (MiB): available: 37659.93, total: 243219.00
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): available: 37659.93, total: 243219.00
	Space in storage for io.netty.native.workdir (MiB): available: 37659.87, total: 243219.00
	Space in storage for java.io.tmpdir (MiB): available: 37659.87, total: 243219.00
	Space in storage for workdir (MiB): available: 37659.87, total: 243219.00
	JVM Flags: 8 total; -Xmx4096M -XX: UnlockExperimentalVMOptions -XX: UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump
	Fabric Mods: 
		fabric-api: Fabric API 0.111.0 1.21.4
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.84 7feeb73304
			fabric-biome-api-v1: Fabric Biome API (v1) 15.0.4 466a140b04
			fabric-block-api-v1: Fabric Block API (v1) 1.0.31 7feeb73304
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.19 7feeb73304
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 2.0.7 7feeb73304
			fabric-client-tags-api-v1: Fabric Client Tags 1.1.27 7feeb73304
			fabric-command-api-v1: Fabric Command API (v1) 1.2.61 f71b366f04
			fabric-command-api-v2: Fabric Command API (v2) 2.2.40 e496eb1504
			fabric-commands-v0: Fabric Commands (v0) 0.2.78 df3654b304
			fabric-content-registries-v0: Fabric Content Registries (v0) 9.1.12 7feeb73304
			fabric-convention-tags-v1: Fabric Convention Tags 2.1.6 7f945d5b04
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.9.6 466a140b04
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.3.6 7feeb73304
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.4.3 9aea556b04
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 22.1.2 466a140b04
			fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.10 7feeb73304
			fabric-entity-events-v1: Fabric Entity Events (v1) 2.0.11 60fccc7604
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 4.0.1 a4eebcf004
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.62 7feeb73304
			fabric-item-api-v1: Fabric Item API (v1) 11.1.12 203e6b2304
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.1.21 7feeb73304
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.56 7feeb73304
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.54 df3654b304
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.5.3 bf2a60eb04
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.31 3f89f5a504
			fabric-loot-api-v3: Fabric Loot API (v3) 1.0.19 203e6b2304
			fabric-message-api-v1: Fabric Message API (v1) 6.0.24 7feeb73304
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 4.1.0 7b6b225304
			fabric-networking-api-v1: Fabric Networking API (v1) 4.3.8 cc0fa2fe04
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 18.0.5 e604fe7f04
			fabric-particles-v1: Fabric Particles (v1) 4.0.13 7feeb73304
			fabric-recipe-api-v1: Fabric Recipe API (v1) 8.0.3 fa62a02304
			fabric-renderer-indigo: Fabric Renderer - Indigo 2.0.1 84404cdd04
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.57 73761d2e04
			fabric-rendering-v1: Fabric Rendering (v1) 10.1.0 7e7568bf04
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 5.0.13 203e6b2304
			fabric-screen-api-v1: Fabric Screen API (v1) 2.0.37 7feeb73304
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.106 7feeb73304
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.32 7feeb73304
			fabric-transfer-api-v1: Fabric Transfer API (v1) 5.4.6 1885ad7404
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.3.2 56e78b9b04
		fabric-language-kotlin: Fabric Language Kotlin 1.13.0 kotlin.2.1.0
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 2.1.0
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 2.1.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 2.1.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 2.1.0
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.26.1
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.9.0
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.9.0
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.6.1
			org_jetbrains_kotlinx_kotlinx-io-bytestring-jvm: kotlinx-io-bytestring-jvm 0.6.0
			org_jetbrains_kotlinx_kotlinx-io-core-jvm: kotlinx-io-core-jvm 0.6.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.7.3
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.7.3
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.7.3
		fabricloader: Fabric Loader 0.16.9
			mixinextras: MixinExtras 0.4.1
		java: OpenJDK 64-Bit Server VM 21
		liquidbounce: LiquidBounce 0.24.1
			com_github_ccbluex_discordipc: DiscordIPC 4.0.0
			com_github_ccbluex_mc-authlib: mc-authlib 1.4.1
			com_github_ccbluex_netty-httpserver: netty-httpserver 2.1.1
			com_google_code_gson_gson: gson 2.10.1
			com_kohlschutter_junixsocket_junixsocket-common: junixsocket-common 2.8.3
			com_kohlschutter_junixsocket_junixsocket-native-common: junixsocket-native-common 2.8.3
			com_squareup_okhttp3_okhttp: okhttp 5.0.0-alpha.14
			com_squareup_okio_okio-jvm: okio-jvm 3.9.0
			com_thealtening_api_api: api 4.1.0
			com_vdurmont_semver4j: semver4j 3.1.0
			commons-codec_commons-codec: commons-codec 1.17.1
			commons-io_commons-io: commons-io 2.16.1
			io_netty_netty-codec-socks: netty-codec-socks 4.1.97
			io_netty_netty-handler: netty-handler 4.1.97
			io_netty_netty-handler-proxy: netty-handler-proxy 4.1.97
			io_netty_netty-resolver: netty-resolver 4.1.97
			io_netty_netty-transport-native-unix-common: netty-transport-native-unix-common 4.1.97
			mcef: MCEF (Minecraft Chromium Embedded Framework) 1.3.2-1.21.4
			net_fabricmc_tiny-mappings-parser: tiny-mappings-parser 0.3.0 build.17
			org_ahocorasick_ahocorasick: ahocorasick 0.6.3
			org_apache_commons_commons-compress: commons-compress 1.27.1
			org_apache_commons_commons-exec: commons-exec 1.3
			org_apache_commons_commons-lang3: commons-lang3 3.16.0
			org_apache_logging_log4j_log4j-api: log4j-api 2.23.1
			org_apache_logging_log4j_log4j-core: log4j-core 2.23.1
			org_apache_tika_tika-core: tika-core 2.9.2
			org_graalvm_js_js-language: js-language 24.0.2
			org_graalvm_polyglot_polyglot: polyglot 24.0.2
			org_graalvm_regex_regex: regex 24.0.2
			org_graalvm_sdk_collections: collections 24.0.2
			org_graalvm_sdk_jniutils: jniutils 24.0.2
			org_graalvm_sdk_nativeimage: nativeimage 24.0.2
			org_graalvm_sdk_word: word 24.0.2
			org_graalvm_shadowed_icu4j: icu4j 24.0.2
			org_graalvm_shadowed_json: json 24.0.2
			org_graalvm_tools_chromeinspector-tool: chromeinspector-tool 24.0.2
			org_graalvm_tools_coverage-tool: coverage-tool 24.0.2
			org_graalvm_tools_dap-tool: dap-tool 24.0.2
			org_graalvm_tools_insight-heap-tool: insight-heap-tool 24.0.2
			org_graalvm_tools_insight-tool: insight-tool 24.0.2
			org_graalvm_tools_lsp-tool: lsp-tool 24.0.2
			org_graalvm_tools_lsp_api: lsp_api 24.0.2
			org_graalvm_tools_profiler-tool: profiler-tool 24.0.2
			org_graalvm_truffle_truffle-api: truffle-api 24.0.2
			org_graalvm_truffle_truffle-compiler: truffle-compiler 24.0.2
			org_graalvm_truffle_truffle-runtime: truffle-runtime 24.0.2
			org_slf4j_slf4j-api: slf4j-api 2.0.10
		meteor-client: Meteor Client 1.21.4-4
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 3.0.11 b1caf1e904
		minecraft: Minecraft 1.21.4
		nochatreports: No Chat Reports 1.21.4-v2.11.0
		reeses-sodium-options: Reese's Sodium Options 1.8.2 mc1.21.4
		sodium: Sodium 0.6.6 mc1.21.4
			fabric-api-base: Fabric API Base 0.4.54 b47eab6b04
			fabric-renderer-api-v1: Fabric Renderer API (v1) 5.0.2 84404cdd04
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.19 7feeb73304
		sodium-extra: Sodium Extra 0.6.1 mc1.21.4
		viafabricplus: ViaFabricPlus 4.0.1
			com_github_oryxel_cubeconverter: CubeConverter 25bc6ce1b1
			com_google_code_findbugs_jsr305: jsr305 3.0.2
			com_viaversion_viaaprilfools-common: viaaprilfools-common 4.0.0
			com_viaversion_viabackwards-common: viabackwards-common 5.2.1
			com_viaversion_vialoader: vialoader 4.0.0
			com_viaversion_viaversion-common: viaversion-common 5.2.1
			de_florianmichael_classic4j: Classic4J 2.1.1-SNAPSHOT
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 6.1.0 9ba00ebb04
			io_jsonwebtoken_jjwt-api: jjwt-api 0.12.6
			io_jsonwebtoken_jjwt-gson: jjwt-gson 0.12.6
			io_jsonwebtoken_jjwt-impl: jjwt-impl 0.12.6
			io_netty_netty-codec-http: netty-codec-http 4.1.116
			net_jodah_expiringmap: expiringmap 0.5.10
			net_lenni0451_commons_httpclient: httpclient 1.6.0
			net_lenni0451_mcping: MCPing 1.4.2
			net_lenni0451_mcstructs-bedrock_forms: forms 1.2.2
			net_lenni0451_mcstructs-bedrock_text: text 1.2.2
			net_lenni0451_reflect: Reflect 1.4.0
			net_raphimc_minecraftauth: MinecraftAuth 4.1.1
			net_raphimc_viabedrock: ViaBedrock 0.0.14-20241220.132943-7
			net_raphimc_vialegacy: ViaLegacy 3.0.7
			org_cloudburstmc_netty_netty-transport-raknet: netty-transport-raknet 1.0.0.CR3-SNAPSHOT
			org_lz4_lz4-pure-java: lz4-pure-java 1.8.0
			viafabricplus-api: ViaFabricPlus API 4.0.1
			viafabricplus-api-legacy: ViaFabricPlus API Legacy 4.0.1
			viafabricplus-visuals: ViaFabricPlus Visuals 4.0.1
	Launched Version: fabric-loader-0.16.9-1.21.4
	Launcher name: LiquidLauncher
	Backend library: LWJGL version 3.3.3-snapshot
	Backend API: NVIDIA GeForce GTX 1650/PCIe/SSE2 GL version 3.2.0 NVIDIA 556.13, NVIDIA Corporation
	Window size: <not initialized>
	GFLW Platform: win32
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Is Modded: Definitely; Client brand changed to 'fabric'
	Universe: 404
	Type: Client (map_client.txt)
	Locale: en_US
	System encoding: Cp1252
	File encoding: UTF-8
	CPU: 8x Intel(R) Core(TM) i5-9300H CPU @ 2.40GHz

-- Meteor Client --

Version: 1.21.4
Build: 4

#@!@# Game crashed! Crash report saved to: #@!@# C:\Users\========\AppData\Roaming\CCBlueX\LiquidLauncher\data\gameDir\nextgen\crash-reports\crash-2025-01-16_20.19.17-client.txt
An error occured:

Process exited with non-zero exit code: -1.
```

### Screenshots

_No response_

--- a/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/gui/MixinPlayerListHud.java
+++ b/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/gui/MixinPlayerListHud.java
@@ -40,9 +40,7 @@ import org.spongepowered.asm.mixin.Shadow;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.ModifyArg;
-import org.spongepowered.asm.mixin.injection.ModifyArgs;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
-import org.spongepowered.asm.mixin.injection.invoke.arg.Args;
 
 import java.util.Comparator;
 import java.util.List;
@@ -145,34 +143,27 @@ public abstract class MixinPlayerListHud {
         }
     }
 
-    @ModifyArgs(method = "render", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/gui/DrawContext;fill(IIIII)V", ordinal = 2))
-    private void hookRenderPlayerBackground(Args args, @Local(ordinal = 13) int w, @Local(ordinal = 0) List<PlayerListEntry> entries) {
-        if (!ModuleBetterTab.INSTANCE.getRunning()) {
-            return;
-        }
+    // ModifyArg breaks lunar compatibility as of 17.1.2025 (minecraft 1.21.4); that's why WrapOperation is used
+    @WrapOperation(method = "render", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/gui/DrawContext;fill(IIIII)V", ordinal = 2))
+    private void hookRenderPlayerBackground(DrawContext instance, int x1, int y1, int x2, int y2, int color, Operation<Void> original, @Local(ordinal = 13) int w, @Local(ordinal = 0) List<PlayerListEntry> entries) {
+        var drawColor = color;
 
         var highlight = ModuleBetterTab.Highlight.INSTANCE;
-        if (!highlight.getRunning()) {
-            return;
-        }
-
-        if (w < entries.size()) {
+        if (ModuleBetterTab.INSTANCE.getRunning() && highlight.getRunning() && w < entries.size()) {
             var entry = entries.get(w);
-            if (highlight.getSelf().getRunning() && Objects.equals(entry.getProfile().getName(), MinecraftClient.getInstance().player.getGameProfile().getName())) {
-                args.set(4, highlight.getSelf().getColor().toARGB());
-                return;
-            }
-
-            if (highlight.getFriends().getRunning() && FriendManager.INSTANCE.isFriend(entry.getProfile().getName())) {
-                args.set(4, highlight.getFriends().getColor().toARGB());
-                return;
-            }
-
             var others = highlight.getOthers();
-            if (others.getRunning() && others.getFilter().isInFilter(entry)) {
-                args.set(4, others.getColor().toARGB());
+
+            //noinspection DataFlowIssue
+            if (highlight.getSelf().getRunning() && Objects.equals(entry.getProfile().getName(), MinecraftClient.getInstance().player.getGameProfile().getName())) {
+                drawColor = highlight.getSelf().getColor().toARGB();
+            } else if (highlight.getFriends().getRunning() && FriendManager.INSTANCE.isFriend(entry.getProfile().getName())) {
+                drawColor = highlight.getFriends().getColor().toARGB();
+            } else if (others.getRunning() && others.getFilter().isInFilter(entry)) {
+                drawColor = others.getColor().toARGB();
             }
         }
+
+        original.call(instance, x1, y1, x2, y2, drawColor);
     }
 
     @ModifyReturnValue(method = "getPlayerName", at = @At("RETURN"))

--- a/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/gui/MixinPlayerListHud.java
+++ b/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/gui/MixinPlayerListHud.java
@@ -37,7 +37,10 @@ import net.minecraft.util.Colors;
 import net.minecraft.util.math.MathHelper;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.Shadow;
-import org.spongepowered.asm.mixin.injection.*;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.ModifyArg;
+import org.spongepowered.asm.mixin.injection.ModifyArgs;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 import org.spongepowered.asm.mixin.injection.invoke.arg.Args;
 
@@ -52,101 +55,73 @@ public abstract class MixinPlayerListHud {
     @Shadow
     protected abstract List<PlayerListEntry> collectPlayerEntries();
 
-    @ModifyConstant(constant = @Constant(longValue = 80L), method = "collectPlayerEntries")
-    private long hookTabSize(long count) {
-        return ModuleBetterTab.INSTANCE.getRunning() ?
-                ModuleBetterTab.Limits.INSTANCE.getTabSize() : count;
+    @WrapOperation(method = "collectPlayerEntries", at = @At(value = "INVOKE", target = "Ljava/util/stream/Stream;limit(J)Ljava/util/stream/Stream;"))
+    private Stream<PlayerListEntry> injectTabSize(Stream<PlayerListEntry> instance, long l, Operation<Stream<PlayerListEntry>> original) {
+        long size = ModuleBetterTab.INSTANCE.getRunning() ? ModuleBetterTab.Limits.INSTANCE.getTabSize() : l;
+        return original.call(instance, size);
     }
 
     @WrapOperation(method = "collectPlayerEntries", at = @At(value = "INVOKE", target = "Ljava/util/stream/Stream;sorted(Ljava/util/Comparator;)Ljava/util/stream/Stream;"))
     private Stream<PlayerListEntry> hookSort(Stream<PlayerListEntry> instance, Comparator<PlayerListEntry> defaultComparator, Operation<Stream<PlayerListEntry>> original) {
-        var sorting = ModuleBetterTab.INSTANCE.getSorting();
-
         var betterTab = ModuleBetterTab.INSTANCE;
-        var running = betterTab.getRunning();
-        var customComparator = sorting.getComparator();
 
-        var comparator = running
-                ? (customComparator != null ? customComparator : defaultComparator)
-                : defaultComparator;
+        var running = betterTab.getRunning();
+        var customComparator = betterTab.getSorting().getComparator();
+
+        var comparator = running ? (customComparator != null ? customComparator : defaultComparator) : defaultComparator;
 
         var playerHider = ModuleBetterTab.PlayerHider.INSTANCE;
-        var hided = running && playerHider.getRunning()
-                ? instance.filter(player -> !playerHider.getFilter().isInFilter(player)) : instance;
+        var hidden = running && playerHider.getRunning() ? instance.filter(player -> !playerHider.getFilter().isInFilter(player)) : instance;
 
-        return original.call(hided, comparator);
+        return original.call(hidden, comparator);
     }
 
-    @ModifyExpressionValue(method = "render", at = @At(
-            value = "FIELD",
-            target = "Lnet/minecraft/client/gui/hud/PlayerListHud;header:Lnet/minecraft/text/Text;",
-            ordinal = 0
-    ))
+    @ModifyExpressionValue(method = "render", at = @At(value = "FIELD", target = "Lnet/minecraft/client/gui/hud/PlayerListHud;header:Lnet/minecraft/text/Text;", ordinal = 0))
     private Text hookHeader(Text original) {
         if (!ModuleBetterTab.INSTANCE.getRunning()) {
             return original;
         }
 
-        return ModuleBetterTab.Visibility.INSTANCE.getHeader() ?
-                original : null;
+        return ModuleBetterTab.Visibility.INSTANCE.getHeader() ? original : null;
     }
 
-    @ModifyExpressionValue(method = "render", at = @At(
-            value = "FIELD",
-            target = "Lnet/minecraft/client/gui/hud/PlayerListHud;footer:Lnet/minecraft/text/Text;",
-            ordinal = 0
-    ))
+    @ModifyExpressionValue(method = "render", at = @At(value = "FIELD", target = "Lnet/minecraft/client/gui/hud/PlayerListHud;footer:Lnet/minecraft/text/Text;", ordinal = 0))
     private Text hookFooter(Text original) {
         if (!ModuleBetterTab.INSTANCE.getRunning()) {
             return original;
         }
 
-        return ModuleBetterTab.Visibility.INSTANCE.getFooter() ?
-                original : null;
+        return ModuleBetterTab.Visibility.INSTANCE.getFooter() ? original : null;
     }
 
-    @ModifyExpressionValue(method = "render", at = @At(
-            value = "FIELD",
-            target = "Lnet/minecraft/client/gui/hud/PlayerListHud$ScoreDisplayEntry;name:Lnet/minecraft/text/Text;"
-    ))
+    @ModifyExpressionValue(method = "render", at = @At(value = "FIELD", target = "Lnet/minecraft/client/gui/hud/PlayerListHud$ScoreDisplayEntry;name:Lnet/minecraft/text/Text;"))
     private Text hookVisibilityName(Text original, @Local(ordinal = 0) PlayerListEntry entry) {
         if (!ModuleBetterTab.INSTANCE.getRunning()) {
             return original;
         }
 
-        return ModuleBetterTab.Visibility.INSTANCE.getNameOnly() ?
-                Text.of(entry.getProfile().getName()) : original;
+        return ModuleBetterTab.Visibility.INSTANCE.getNameOnly() ? Text.of(entry.getProfile().getName()) : original;
 
     }
 
-    @ModifyExpressionValue(method = "render", at = @At(
-            value = "INVOKE",
-            target = "Lnet/minecraft/client/gui/hud/PlayerListHud;getPlayerName(Lnet/minecraft/client/network/PlayerListEntry;)Lnet/minecraft/text/Text;"
-    ))
+    @ModifyExpressionValue(method = "render", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/gui/hud/PlayerListHud;getPlayerName(Lnet/minecraft/client/network/PlayerListEntry;)Lnet/minecraft/text/Text;"))
     private Text hookWidthVisibilityName(Text original, @Local(ordinal = 0) PlayerListEntry entry) {
         if (!ModuleBetterTab.INSTANCE.getRunning()) {
             return original;
         }
 
-        return ModuleBetterTab.Visibility.INSTANCE.getNameOnly() ?
-                Text.of(entry.getProfile().getName()) : original;
+        return ModuleBetterTab.Visibility.INSTANCE.getNameOnly() ? Text.of(entry.getProfile().getName()) : original;
     }
 
     @Inject(method = "render", at = @At(value = "INVOKE", target = "Ljava/lang/Math;min(II)I", shift = At.Shift.BEFORE))
-    private void hookTabColumnHeight(CallbackInfo ci, @Local(ordinal = 5) LocalIntRef o, @Local(ordinal = 6)LocalIntRef p) {
+    private void hookTabColumnHeight(CallbackInfo ci, @Local(ordinal = 5) LocalIntRef o, @Local(ordinal = 6) LocalIntRef p) {
         if (!ModuleBetterTab.INSTANCE.getRunning()) {
             return;
         }
 
-        int totalPlayers = collectPlayerEntries().size();
-        int columns = 1;
-        int rows = totalPlayers;
-
-        while (rows > ModuleBetterTab.Limits.INSTANCE.getHeight()) {
-            columns++;
-            rows = (totalPlayers + columns - 1) / columns;
-        }
-
+        int playerCount = collectPlayerEntries().size();
+        int columns = MathHelper.ceil((double) playerCount / ModuleBetterTab.Limits.INSTANCE.getHeight());
+        int rows = MathHelper.ceil((double) playerCount / columns);
         o.set(rows);
         p.set(columns);
     }
@@ -183,25 +158,19 @@ public abstract class MixinPlayerListHud {
 
         if (w < entries.size()) {
             var entry = entries.get(w);
-            if (highlight.getSelf().getRunning()) {
-                if (Objects.equals(entry.getProfile().getName(), MinecraftClient.getInstance().player.getGameProfile().getName())) {
-                    args.set(4, highlight.getSelf().getColor().toARGB());
-                    return;
-                }
+            if (highlight.getSelf().getRunning() && Objects.equals(entry.getProfile().getName(), MinecraftClient.getInstance().player.getGameProfile().getName())) {
+                args.set(4, highlight.getSelf().getColor().toARGB());
+                return;
             }
 
-            if (highlight.getFriends().getRunning()) {
-                if (FriendManager.INSTANCE.isFriend(entry.getProfile().getName())) {
-                    args.set(4, highlight.getFriends().getColor().toARGB());
-                    return;
-                }
+            if (highlight.getFriends().getRunning() && FriendManager.INSTANCE.isFriend(entry.getProfile().getName())) {
+                args.set(4, highlight.getFriends().getColor().toARGB());
+                return;
             }
 
             var others = highlight.getOthers();
-            if (others.getRunning()) {
-                if (others.getFilter().isInFilter(entry)) {
-                    args.set(4, others.getColor().toARGB());
-                }
+            if (others.getRunning() && others.getFilter().isInFilter(entry)) {
+                args.set(4, others.getColor().toARGB());
             }
         }
     }
@@ -209,11 +178,9 @@ public abstract class MixinPlayerListHud {
     @ModifyReturnValue(method = "getPlayerName", at = @At("RETURN"))
     private Text modifyPlayerName(Text original, PlayerListEntry entry) {
         if (ModuleAntiStaff.UsernameCheck.INSTANCE.shouldShowAsStaffOnTab(entry.getProfile().getName())) {
-            return original.copy().append(
-                    Text.literal(" - (Staff)")
-                            .withColor(Colors.LIGHT_RED)
-            );
+            return original.copy().append(Text.literal(" - (Staff)").withColor(Colors.LIGHT_RED));
         }
+
         return original;
     }
 

--- a/src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/misc/ModuleBetterTab.kt
+++ b/src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/misc/ModuleBetterTab.kt
@@ -1,3 +1,21 @@
+/*
+ * This file is part of LiquidBounce (https://github.com/CCBlueX/LiquidBounce)
+ *
+ * Copyright (c) 2015 - 2025 CCBlueX
+ *
+ * LiquidBounce is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ */
 package net.ccbluex.liquidbounce.features.module.modules.misc
 
 import net.ccbluex.liquidbounce.config.types.Configurable
@@ -16,16 +34,6 @@ import net.minecraft.text.Text
  * @since 12/28/2024
  **/
 object ModuleBetterTab : ClientModule("BetterTab", Category.MISC) {
-
-    init {
-        treeAll(
-            Limits,
-            Visibility,
-            Highlight,
-            AccurateLatency,
-            PlayerHider
-        )
-    }
 
     val sorting by enumChoice("Sorting", Sorting.VANILLA)
 
@@ -64,6 +72,17 @@ object ModuleBetterTab : ClientModule("BetterTab", Category.MISC) {
     object PlayerHider : ToggleableConfigurable(ModuleBetterTab, "PlayerHider", false) {
         val filter = tree(PlayerFilter())
     }
+
+    init {
+        treeAll(
+            Limits,
+            Visibility,
+            Highlight,
+            AccurateLatency,
+            PlayerHider
+        )
+    }
+
 }
 
 class PlayerFilter: Configurable("Filter") {

